terraform/vars/tvm-ci-prod.auto.tfvars

```diff
@@ -38,6 +38,15 @@ autoscaler_types = {
     on_demand_percentage_above_base_capacity = 100
     on_demand_base_capacity                  = 0
   }
+  "Prod-Autoscaler-Jenkins-CPU-SPOT" = {
+    image_family                             = "jenkins-stock-agent-x64"
+    agent_instance_type                      = "c4.4xlarge"
+    labels                                   = "CPU-SPOT"
```

Comment:
Doesn't need to be part of this PR but I think we should flip the CPU/CPU-SMALL to discourage the use of CPU to CPU-LARGE/CPU instead